polaris-core/src/main/java/org/apache/polaris/core/config/BehaviorChangeConfiguration.java

```diff
@@ -59,4 +59,13 @@ protected BehaviorChangeConfiguration(
           .description("Whether or not to use soft values in the entity cache")
           .defaultValue(false)
           .buildBehaviorChangeConfiguration();
+
+  public static final BehaviorChangeConfiguration<Boolean> TABLE_OPERATIONS_COMMIT_UPDATE_METADATA =
+      PolarisConfiguration.<Boolean>builder()
+          .key("TABLE_OPERATIONS_COMMIT_UPDATE_METADATA")
+          .description(
+              "If true, BasePolarisTableOperations should cache metadata that has been committed"
```

Comment:
based on this description I'd propose to rename the property to `CACHE_COMMITTED_METADATA`.

Reply:
Good callout, there is a mismatch between the description and the name of this flag after my rebase. I will adjust the description for now but let me know if you think we need further changes